commands/mail.js

```diff
@@ -256,28 +298,57 @@ module.exports = {
     request.method = "POST";
     request.url = "/v3/mail/send";
 
-    log.sys(`stringify request made by client: ${JSON.stringify(client.createRequest(request))}`);
-
+    // create a shallow copy of the request for logging
+    const clientRequest = { ...client.createRequest(request) }; // Shallow copy
+    unsetField(clientRequest, "Authorization"); // remove sensitive fileds ("Authorization")
+    log.sys(`stringify request made by client: ${JSON.stringify(clientRequest)}`);
```

Comment:
Should this be log.debug and only shown in debug mode?

Reply:
@tlawrie log.sys it was until now, but we can change it to log.debug.